README.md

```diff
@@ -9,8 +9,8 @@ A Go library for generating MPEG-DASH manifests.
 This library uses [Glide](https://github.com/Masterminds/glide) to manage it's dependencies. Please refer to the Glide documentation to see how to install glide.
 
 ```bash
-mkdir -p $GOPATH/zencoder
-cd $GOPATH/zencoder
+mkdir -p $GOPATH/github.com/zencoder
```

Comment:
![](https://i.imgur.com/STPmXQU.gif)

What idiot review-passed this?! :stuck_out_tongue: